string sensor_name
float64 temperature